src/homeWorkSecond/CheckingExpression.java

```diff
@@ -0,0 +1,13 @@
+package homeWorkSecond;
```

Comment:
CS

Reply:
The name of my package is wrong?
how better? "homework.second" or "womeworkSecond"??